app/renderer/css/tagify.css

```diff
@@ -0,0 +1,2 @@
nslatex(6px)}@supports (-moz-appearance:none){.tagify__input:focus::before{display:none}}@media all and (-ms-high-contrast:none),(-ms-high-contrast:active){.tagify__input:focus::before{display:none}}@supports (-ms-ime-align:auto){.tagify__input:focus::before{display:none}}.tagify__input:focus:empty::before{transition:.2s ease-out;opacity:.3;transform:none}@supports (-moz-appearance:none){.tagify__input:focus:empty::before{content:unset}}.tagify__input::before{content:attr(data-placeholder);position:absolute;top:0;bottom:0;height:1em;line-height:1em;margin:auto 0;z-index:1;color:#000;color:var(--placeholder-color);white-space:nowrap;pointer-events:none;opacity:0}.tagify--mix .tagify__input::before{position:static;line-height:inherit}.tagify__input::after{content:attr(data-suggest);display:inline-block;white-space:pre;color:#000;opacity:.3;pointer-events:none;max-width:100px}.tagify__input .tagify__tag{margin:0}.tagify__input .tagify__tag>div{padding-top:0;padding-bottom:0}.tagify--mix{line-height:1.7}.tagify--mix .tagify__input{padding:5px;margin:0;width:100%;height:100%;line-height:inherit}.tagify--mix .tagify__input::before{height:auto}.tagify--mix .tagify__input::after{content:none}.tagify--select::after{content:'>';opacity:.5;position:absolute;top:50%;right:0;bottom:0;font:16px monospace;line-height:8px;height:8px;pointer-events:none;transform:translate(-150%,-50%) scaleX(1.2) rotate(90deg);transition:.2s ease-in-out}.tagify--select[aria-expanded=true]::after{transform:translate(-150%,-50%) rotate(270deg) scaleY(1.2)}.tagify--select .tagify__tag{position:absolute;top:0;right:1.8em;bottom:0}.tagify--select .tagify__tag div{display:none}.tagify--select .tagify__input{width:100%}.tagify--invalid{--tags-border-color:#D39494}.tagify__dropdown{position:absolute;z-index:9999;transform:translateY(1px);overflow:hidden}.tagify__dropdown[placement=top]{margin-top:0;transform:translateY(-2px)}.tagify__dropdown[placement=top] .tagify__dropdown__wrapper{border-top-width:1px;border-bottom-width:0}.tagify__dropdown--text{box-shadow:0 0 0 3px rgba(var(--tagify-dd-color-primary),.1);font-size:.9em}.tagify__dropdown--text .tagify__dropdown__wrapper{border-width:1px}.tagify__dropdown__wrapper{max-height:300px;overflow:hidden;background:#fff;background:var(--tagify-dd-bg-color);border:1px solid #3595f6;border-color:var(--tagify-dd-color-primary);border-top-width:0;box-shadow:0 2px 4px -2px rgba(0,0,0,.2);transition:.25s cubic-bezier(0,1,.5,1)}.tagify__dropdown__wrapper:hover{overflow:auto}.tagify__dropdown--initial .tagify__dropdown__wrapper{max-height:20px;transform:translateY(-1em)}.tagify__dropdown--initial[placement=top] .tagify__dropdown__wrapper{transform:translateY(2em)}.tagify__dropdown__item{box-sizing:inherit;padding:.3em .5em;margin:1px;cursor:pointer;border-radius:2px;position:relative;outline:0}.tagify__dropdown__item--active{background:#3595f6;background:var(--tagify-dd-color-primary);color:#fff}.tagify__dropdown__item:active{filter:brightness(105%)}
```

Comment:
I assume this is copied and pasted from the `@yaireo/tagify` package. We should never copy and paste dependencies; we should load them from `node_modules`. One way to do that would be to add the stylesheet in JavaScript from `require.resolve("@yaireo/tagify/dist/tagify.css")`. A more general solution would be to build the app with webpack and html-webpack-plugin, or some other bundler.

Reply:
> One way to do that would be to add the stylesheet in JavaScript from `require.resolve("@yaireo/tagify/dist/tagify.css")`.

By this are you suggesting to add something like this in `preference.html` - 
```js
		let tagifyCSS = document.createElement("link");
		tagifyCSS.rel = "stylesheet";
		tagifyCSS.type = "text/css";
		tagifyCSS.href = require.resolve("@yaireo/tagify/dist/tagify.css");
		document.querySelector("head").appendChild(tagifyCSS);
```

Reply:
@abhigyank the snippet can be shortened by assigning id to existing tagify \<link\>
```ts
document.querySelector('#tagifyCSS').href = require.resolve('@yaireo/tagify/dist/tagify.css');
```